dss-pades-pdfbox/src/main/java/eu/europa/esig/dss/pdf/pdfbox/visible/nativedrawer/SignatureFieldDimensionAndPositionBuilder.java

```diff
@@ -197,10 +198,13 @@ private void assignImageBoundaryBox() {
 				break;
 			}
 
-			dimensionAndPosition.setTextWidth(textWidth);
-			dimensionAndPosition.setTextHeight(textHeight);
-			Integer imageDpi = imageParameters.getImage() != null ? imageParameters.getDpi() : DEFAULT_DPI;
```

Comment:
The logic of this particular line is lost with this change but the fact that it was used in only one of the padding calculations and also that it looks like a very uncommon scenario led me to consider that it was probably a hidden bug more than a very special corner case.